.github/workflows/tii-mesh-com.yaml

```diff
@@ -43,6 +41,6 @@ jobs:
         with:
           context: .
           file: ./modules/mesh_com/Dockerfile
-          push: ${{ github.event_name != 'pull_request' }}
+          push: true
```

Comment:
Should this change be reverted to avoid unnecessary big amount of images pushed into ghcr during some feature testing/developing/reviewing?

Reply:
Maybe we go with this change now. Need more discussion with bigger audience what is the policy regarding image push to ghcr